AutomatticTracks/lint-baseline.xml

```diff
@@ -1,5 +1,148 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<issues format="5" by="lint 4.0.2" client="gradle" variant="all" version="4.0.2">
+<issues format="5" by="lint 4.0.2" client="gradle" variant="release" version="4.0.2">
+
+    <issue
+        id="Range"
+        message="Value must be ≥ 0"
+        errorLine1="                    idToDelete.add(cursor.getLong(cursor.getColumnIndex(&quot;event_id&quot;)));"
```

Comment:
Those are new lint issues but about an old, not edited for years code. So I think it's okay to add them to the baseline in the scope of this PR.

Reply:
👋 @wzieba !

Suggestion (💡): I understand why you suppress those, this is not a big deal, but I actually recommend resolving them, as the resolution is easy. Also, we already did exactly that for WPAndroid and WCAndroid. So, please consider just replacing `cursor.getColumnIndex(...)` with `cursor.getColumnIndexOrThrow(...)` on all 13 occurrences of those within the `EventTable.java` file. That would be it. Then, this `lint-baseline.xml` file will be kept short. Wdyt? 🤔